--- crudo/empresas.yaml
+++ content/ (empresas)
@@ -4599,9 +4599,6 @@
           el PPI calculada por un tercero.
         periodo: "2024"
         producto: supergás
-  procedencia:
-    correccion: 2026-09-08-ancap-segmentos-paridad-hitos
-    tipo: correccion
   que_hace: ANCAP administra y explota el monopolio de la importación y refinación de petróleo, y de
     la importación y exportación de combustibles líquidos, semilíquidos y gaseosos, con excepción de
     la importación de gas natural. Sus principales actividades son la importación y refinación de
