--- crudo/empresas.yaml
+++ content/ (empresas)
@@ -2830,9 +2830,6 @@
           el PPI calculada por un tercero.
         periodo: "2024"
         producto: supergás
-  procedencia:
-    correccion: 2026-09-07-ancap-ficha-y-reexpresion-2020
-    tipo: correccion
   que_hace: ANCAP administra y explota el monopolio de la importación y refinación de petróleo, y de
     la importación y exportación de combustibles líquidos, semilíquidos y gaseosos, con excepción de
     la importación de gas natural. Sus principales actividades son la importación y refinación de
