--- crudo/politicos.yaml
+++ content/ (politicos)
@@ -121,6 +121,8 @@
   nombre: Raúl Fernando Sendic Rodríguez
   nombre_corto: Raúl Sendic
   partido: Frente Amplio
+  revision:
+    tier: publicado
   wikidata: Q6101178
 - alias:
     - Lucía Topolansky
@@ -218,6 +220,8 @@
   nombre: Lucía Topolansky Saavedra
   nombre_corto: Lucía Topolansky
   partido: Frente Amplio
+  revision:
+    tier: publicado
   wikidata: Q977078
 - alias:
     - Beatriz Argimón
@@ -278,6 +282,8 @@
   nombre: Beatriz Argimón Cedeira
   nombre_corto: Beatriz Argimón
   partido: Partido Nacional
+  revision:
+    tier: publicado
   wikidata: Q5724101
 - alias:
     - Carolina Cosse
@@ -357,4 +363,6 @@
   nombre: Ana Carolina Cosse Garrido
   nombre_corto: Carolina Cosse
   partido: Frente Amplio
+  revision:
+    tier: publicado
   wikidata: Q16543727
