--- crudo/politicos.yaml
+++ content/ (politicos)
@@ -108,6 +108,8 @@
   nombre: Luis Antonio Hierro López
   nombre_corto: Luis Hierro López
   partido: Partido Colorado
+  revision:
+    tier: publicado
   wikidata: Q961132
 - alias:
     - Rodolfo Nin Novoa
@@ -241,6 +243,8 @@
   nombre: Rodolfo Nin Novoa
   nombre_corto: Nin Novoa
   partido: Frente Amplio
+  revision:
+    tier: publicado
   wikidata: Q1344343
 - alias:
     - Danilo Astori
@@ -357,4 +361,6 @@
   nombre: Danilo Ángel Astori Saragosa
   nombre_corto: Danilo Astori
   partido: Frente Amplio
+  revision:
+    tier: publicado
   wikidata: Q51983
